### DISCUSSION TITLE
Why does Company pop-up not align over faces with different height?

### DISCUSSION THREAD

**Discussion by protesilaos** [company-mode/company-mode#1172] [Q&A] [answered]
Hello and thank you for your numerous contributions!

The following is a variant of [issue 299](https://github.com/company-mode/company-mode/issues/299), but I thought it may be better to post it on its own.

* * *

Please see attached screenshot for the problem.

![Screenshot from 2020-09-21 06-40-57](https://user-images.githubusercontent.com/12828033/93739640-719c9a00-fbf1-11ea-838b-49080c71d4a8.png)

In an `emacs -Q` session configured to run `company-mode`, follow these steps to reproduce the issue:

+ C-x C-f /tmp/test-company.org
+ Insert some common text:

```
Testing phenom

* That is a test
* This is a test

Some words: phenomenal, phenomenality, phenomenology, phenomenon
```

* Try to complete "phenom" on the first line.  The company pop-up displays correctly.

* Now change the height of the headings by evaluating this:

```elisp
(set-face-attribute 'org-level-1 nil :height 2.0)
```

* Try again to complete "phenom".  You should notice the misalignment of the pop-ups.  It only occurs over elements that have a `:height` other than the default (i.e. also for smaller values than `1.0`).

**Comment by dgutov** [comment, marked as answer]
Hi!

This is more or less the same issue, and the problem is that is bumps against the limitation of overlay-based popup rendering.

The advice is the same: install a frontend that doesn't use overlays. Such as https://github.com/tumashu/company-posframe/.

**Comment by protesilaos** [comment]
Thank you for the prompt reply!
